### DISCUSSION TITLE
How do I make the camera follow and orbit with a meshs direction

### DISCUSSION THREAD

**Discussion by Schwagsone** [ramokz/phantom-camera#243] [Question / Help Wanted]
Having a hard time figuring it out. I have a sports game where the camera needs to be facing the direction the player is moving.

**Comment by ramokz** [comment]
Sounds like that would be the [Third Person Follow](https://phantom-camera.dev/follow-modes/third-person) mode, which uses a SpringArm3D for physics collision?
You would just need to apply the rotation logic to the camera manually. There is some example code in both the project and in the link above.
